### PR TITLE
Update how-to-enable-dmarc-reporting-for-microsoft-online-email-routi…

### LINKED ISSUE #12272: Missing detail on DMARC value
"Add your specific DMARC value." is not very descriptive. Where do I find that? How to I generate/build that?

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 521a61f4-edfa-5aff-598e-6a63480e4a6e
* Version Independent ID: 521a61f4-edfa-5aff-598e-6a63480e4a6e
* Content: [How to enable DMARC Reporting for Microsoft Online Email Routing Address (MOERA) and parked Domains](https://learn.microsoft.com/en-us/microsoft-365/security/office-365-security/step-by-step-guides/how-to-enable-dmarc-reporting-for-microsoft-online-email-routing-address-moera-and-parked-domains?view=o365-worldwide)
* Content Source: [microsoft-365/security/office-365-security/step-by-step-guides/how-to-enable-dmarc-reporting-for-microsoft-online-email-routing-address-moera-and-parked-domains.md](https://github.com/MicrosoftDocs/microsoft-365-docs/blob/public/microsoft-365/security/office-365-security/step-by-step-guides/how-to-enable-dmarc-reporting-for-microsoft-online-email-routing-address-moera-and-parked-domains.md)
* Service: **microsoft-365-security**
* Sub-service: **mdo**
* GitHub Login: @MSFTTracyP
* Microsoft Alias: **tracyp**

### PR DESCRIPTION
**Pull request by msbemba**
…ng-address-moera-and-parked-domains.md

Added more information on how to create DMARC

Fixes https://github.com/MicrosoftDocs/microsoft-365-docs/issues/12272

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/security/office-365-security/step-by-step-guides/how-to-enable-dmarc-reporting-for-microsoft-online-email-routing-address-moera-and-parked-domains.md [hunk 1]
```diff
@@ -48,7 +48,7 @@ This guide is designed to help you configure DMARC for domains not covered by th
 1. Select **+ Add record**.
 1. A flyout will appear on the right. Ensure that the selected Type is **TXT (Text)**.
 1. Add `_dmarc` as **TXT name**.
-1. Add your specific DMARC value.
+1. Add your specific DMARC value. For more information check [how to Form the DMARC TXT record for your domain](/microsoft-365/security/office-365-security/email-authentication-dmarc-configure#step-4-form-the-dmarc-txt-record-for-your-domain)
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/step-by-step-guides/how-to-enable-dmarc-reporting-for-microsoft-online-email-routing-address-moera-and-parked-domains.md]:
```suggestion
1. Add your specific DMARC value. For more information, see [Form the DMARC TXT record for your domain](/microsoft-365/security/office-365-security/email-authentication-dmarc-configure#step-4-form-the-dmarc-txt-record-for-your-domain).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@MSFTTracyP Please merge this PR. Thanks.

**Pull request merged**